WpfMath/Controls/VisualContainerElement.cs

```diff
@@ -1,61 +1,61 @@
-    {
-        private DrawingVisual visual;
-
-        public VisualContainerElement()
-            : base()
-        {
-            this.visual = null;
-        }
-
-        public DrawingVisual Visual
-        {
-            get { return this.visual; }
-            set
-            {
-                RemoveVisualChild(this.visual);
-                this.visual = value;
-                AddVisualChild(this.visual);
-
-                InvalidateMeasure();
-                InvalidateVisual();
-            }
-        }
-
-        protected override int VisualChildrenCount
-        {
-            get { return 1; }
-        }
-
-        protected override Visual GetVisualChild(int index)
-        {
-            return this.visual;
-        }
-
-        protected override Size MeasureOverride(Size availableSize)
-        {
-            if (this.visual != null)
-                return this.visual.ContentBounds.Size;
-            return base.MeasureOverride(availableSize);
-        }
-
-        protected override Size ArrangeOverride(Size finalSize)
-        {
-            return base.ArrangeOverride(finalSize);
-        }
-
-        protected override void OnVisualChildrenChanged(DependencyObject visualAdded, DependencyObject visualRemoved)
-        {
-            base.OnVisualChildrenChanged(visualAdded, visualRemoved);
-        }
-    }
-}
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Windows;
+using System.Windows.Media;
+
+namespace WpfMath.Controls
+{
+    public class VisualContainerElement : FrameworkElement
+    {
+        private DrawingVisual visual;
+
+        public VisualContainerElement()
+            : base()
+        {
+            this.visual = null;
+        }
+
+        public DrawingVisual Visual
+        {
+            get { return this.visual; }
+            set
+            {
+                RemoveVisualChild(this.visual);
+                this.visual = value;
+                AddVisualChild(this.visual);
+
+                InvalidateMeasure();
+                InvalidateVisual();
+            }
+        }
+
+        protected override int VisualChildrenCount
+        {
+            get { return 1; }
+        }
+
+        protected override Visual GetVisualChild(int index)
+        {
+            return this.visual;
+        }
+
+        protected override Size MeasureOverride(Size availableSize)
+        {
+            if (this.visual != null)
+                return this.visual.ContentBounds.Size;
+            return base.MeasureOverride(availableSize);
+        }
+
+        protected override Size ArrangeOverride(Size finalSize)
+        {
+            return base.ArrangeOverride(finalSize);
+        }
+
+        protected override void OnVisualChildrenChanged(DependencyObject visualAdded, DependencyObject visualRemoved)
+        {
+            base.OnVisualChildrenChanged(visualAdded, visualRemoved);
+        }
+    }
+}
```

Comment:
Something weird has been happened with my diff on this file. Probably line ending conversion. Nothing wrong with that, but in that case, we should accurately check whether the new line endings are the right ones (we use `\n` here, in the repository).